jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-test.yaml

```diff
@@ -13,6 +13,10 @@
           name: 'installationGitBranch'
           default: 'master'
           description: '[REQUIRED] The name of the git branch to be used for installing Integreatly'
+      - string:
+          name: 'clusterName'
+          default: 'qe-master'
+          description: '[REQUIRED] The name of the newly provisioned cluster on skunkhenry domain.'    
```

Comment:
What's the reason for adding this, is it due to some cert issue or ......?

Reply:
Well we want to run more things nightly (e.g the MSB integration tests) so we think it's better to use a different name than "master" because having "master" everywhere could mean pipeline clashing problems in the future.

Reply:
Can you add some details of why we are doing this to the PR description, link to Jira maybe?

There are issues with using the branch name like we are and we do need to look into using a different naming scheme by default, but the changes you have made to this will cause every single cluster to be called "qe-master" unless you override it in every job that is calling it.

A better option for this would be to leave it using the branch name for now, but check the clusterName param and set it to that value if it's not blank, that way everything will still work as normal but allow you to override it in your nightly jobs.

Do you have details of what exactly you are planning re. nightly jobs? I was under the impression you would have a single nightly job that ran all tests, so not sure where this issue came from about needing several clusters at once.

Reply:
JIRA is https://issues.jboss.org/browse/INTLY-2273. I am failing to see the difference in name "master" vs "qe-master". I am not changing anything in branch naming AFAIK. 

Yes this is our primary pipeline (running all the tests). So maybe we can just change the clusterName in the MSB tests pipeline and keep this as is. 
I am not aware of any further plans for additional nightly pipelines (using tower tooling).

Reply:
+1 Using branch as clusterName is not an optimal approach and needs some adjustments
+1 not having `qe-admin` hard-coded as default here - it should only be used in one of our nightly pipelines, not in this job in general

There will be more nightly jobs than just one single nightly job (MSB integration test was mentioned as example). Also, we want to use this for testing per PR potentially. Each pipeline that (eventually) creates cluster should have cluserName as input parameter. I am ok with defaulting to branch name. So we can specify some clusterName that is unlikely to be used by somebody else in our nightly pipeline.

Reply:
OK I tried to update it accordingly. Need to check if it works...

Reply:
@mikenairn do you have any concerns about this PR?